### DISCUSSION TITLE
How to get local git user name

### DISCUSSION THREAD

**Discussion by justforuse** [ohmyzsh/ohmyzsh#11128] [Help] [answered]
I have set a specific git user name and email for a git repo, but the `git_user_name` alway return the global git user name, is there any way to get the local git user name?

**Comment by tobih7** [comment, marked as answer]
You're looking for `git config --local user.name` and `git config --local user.email`.
But this is not the right place to ask git related questions.

**Reply by justforuse** [reply]
Maybe I confuse the variable name, I found `$(git_current_user_name)` is what I want.
